index.php

```diff
@@ -39,6 +39,16 @@
 $merge_version = "1.8.23";
 $version_code = 1823;
 
+// MyBB Group constants. Used for better readability
+// We need them here to make 100% sure they're defined when creating our classes
+define("MYBB_GUESTS",		1);
+define("MYBB_REGISTERED",	2);
+define("MYBB_SMODS",		3);
+define("MYBB_ADMINS",		4);
+define("MYBB_AWAITING",		5);
+define("MYBB_MODS",			6);
+define("MYBB_BANNED",		7);
```

Comment:
I know this is copied and pasted from below, but the spacing looks really weird. IMO it should just be 1 space for all values:

```suggestion
define("MYBB_GUESTS", 1);
define("MYBB_REGISTERED", 2);
define("MYBB_SMODS", 3);
define("MYBB_ADMINS", 4);
define("MYBB_AWAITING", 5);
define("MYBB_MODS", 6);
define("MYBB_BANNED", 7);
```

Reply:
Yeah that was a cut and paste of old code. I was trying to align them using spaces but end up reverting to the original using tabs.

Reply:
Oh I should just commit your suggestion here and that'd save a edit and push from my local repo. Being silly again 🤣